clang/src/fileio.s

```diff
@@ -6,20 +6,41 @@ ptr1:
         .ds.b 1 ;; 1 byte
 ptr2:
         .ds.b 1
+tmp1:
+        .ds.b 1
 tmp2:
         .ds.b 1
 tmp3:
         .ds.b 1
+sp:
+        .ds.b 1
 
-.section code
+    .section code,"a"
```

Comment:
Just curious what the "a" is for?

Reply:
That was a critical fix as the assembler otherwise didn't complain about the missing symbols. "a" means "allocatable", see [the llvm wiki](https://llvm-mos.org/wiki/Frequently_asked_questions#Why_isn.27t_the_linker_placing_my_sections.3F).